tests/unit/repo/experiments/test_utils.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+
+from dvc.repo.experiments.base import EXPS_NAMESPACE, ExpRefInfo
+from dvc.repo.experiments.utils import resolve_exp_ref
+
+
+@pytest.fixture
+def git_upstream(scm, make_tmp_dir, name="origin"):
+    remote_dir = make_tmp_dir("git-remote", scm=True)
+    url = f"file://{remote_dir.resolve().as_posix()}"
+    scm.gitpython.repo.create_remote(name, url)
+    remote_dir.remote = name
+    remote_dir.url = url
+    return remote_dir
```

Comment:
Why is this fixture needed? As far as I can tell it's the same as the existing `git_upstream` fixture?

If the issue is just fixture scope/availability, you can move the git fixtures from `tests/func/experiments/conftest` into the global `tests/dir_helpers` module.

(Or just add the appropriate import into `tests/conftest.py`)